src/coreclr/jit/hwintrinsic.h

```diff
@@ -57,6 +57,7 @@ enum HWIntrinsicCategory : uint8_t
     // These are Arm64 that share some features in a given category (e.g. immediate operand value range)
     HW_Category_ShiftLeftByImmediate,
     HW_Category_ShiftRightByImmediate,
+    HW_Category_RotateByImmediate,
```

Comment:
Why can't we just use `HW_Category_ShiftRightByImmediate`?

Reply:
> in future.

let's introduce the new flag when there is a need. For now, just reuse `HW_Category_ShiftRightByImmediate`